src/web/services/search.service.spec.ts

```diff
@@ -185,24 +245,36 @@ describe('SearchService', () => {
   });
 
   it('should join students accurately when calling as admin', () => {
-    const result: StudentAccountSearchResult = service.joinAdminStudent(
+    const resultA: StudentAccountSearchResult = service.joinAdminStudent(
```

Comment:
We need to clearly indicate what is being tested for else other's won't know what to check for when the test fails.

Let's split this into two separate testcases using the `it` method

Reply:
I considered that option but I think this is exactly testing whether the student information is joint accurately (one field of `StudentAccountSearchResult`).

Reply:
Then what is `resultA` and `resultB`?

Reply:
Two test cases.

Reply:
Yes, so they should be in separate testcases?

Reply:
Okk I'll separate them into two cases.